.gitignore

```diff
@@ -7,3 +7,4 @@
 /laminas-mkdoc-theme/
 /phpunit.xml
 /vendor/
+.idea
```

Comment:
Please revert; this is something to add to your user global git ignore rules.

Reply:
I updated the code.